scrc/dataset_creation/judgment_dataset_creator.py

```diff
@@ -30,12 +30,15 @@ def __init__(self, config: dict, debug: bool = True):
         self.make_single_label = True
         self.labels = ['label']
         self.start_years = {Split.TRAIN.value: 1970, Split.VALIDATION.value: 2016, Split.TEST.value: 2018,
-                            Split.SECRET_TEST.value: 2020}
+                            Split.SECRET_TEST.value: 2023}
```

Comment:
I think you can add the secret test start year to the dataset creator already as default so that we can make sure to stop at 2022 for the moment

Reply:
Alright.